### ISSUE TITLE
TypeScript support

### ISSUE THREAD

**Issue by zunder** [shoutem/ui#285]
Do you plan to support TypeScript definitions with this project?

**Comment by Goldenstarc** [comment]
its good idea, i plan to use it with reactxp and ts is essential  ! 
just like native base

**Comment by qiaoyan** [comment]
Test 1

**Comment by qiaoyan** [comment]
Test 2

**Comment by qiaoyan** [comment]
Test 3

**Comment by qiaoyan** [comment]
Test 1

**Comment by arminyahya** [comment]
Hi
I wanna to make PR and add type declaration for library
@Vladimir-Vdovic

**Comment by bearkfear** [comment]
Its beautiful. But withtout ts is not a option

**Comment by Pranit-Harekar** [comment]
Yes, please add typings. This is a really great library just lacks TypeScript support!!

**Comment by YassineGherbi** [comment]
It has been a while, has there been any progress on this? If not, can I contribute to the project to support TS

**Comment by gaffarmalik** [comment]
I'm also hoping this library will use typescript one day
